index.js

```diff
@@ -39,7 +39,13 @@ module.exports = function(content) {
       JSON.stringify(fileName) +
       ");" +
       "try { global.process.dlopen(module, filePath); } " +
-      "catch(exception) { throw new Error('Cannot open ' + filePath + ': ' + exception); };"
+      "catch(exception) { " +
+      "try { " +
+      "const filePath2 = path.resolve(__dirname, '..', '..', 'app.asar', " +
```

Comment:
Then there won't be confusing `filePath2` variable name (the numerical suffix part)

Reply:
Also, the package could be named differently than `app.asar` (I would put it to the configuration too with default `app.asar`)